spec/lib/twingly/url/hasher_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+SimpleCov.command_name(File.basename(__FILE__, ".rb"))
```

Comment:
Sure, I should have mentioned it. I has to do that we have split our tests over several different rake tasks and need to merge the results. I got it from https://github.com/colszowka/simplecov#merging-results. I tried doing `SimpleCov.command_name = "PID#{Process.pid}"` in `spec_helper.rb`, so we shouldn't have to add something to every spec file, but it didn't work well. It saved the results from every run, so it didn't discover regressions in coverage.

The command name can be seen in the footer in report:

> Generated by simplecov v0.12.0 and simplecov-html v0.10.0
using hasher_spec, null_url_spec, url_spec, utilities_spec